packages/devextreme/js/__internal/ui/selection/m_selection.strategy.standard.ts

```diff
@@ -271,6 +271,12 @@ export default class StandardStrategy extends SelectionStrategy {
     const selectionDeferred = Deferred();
 
     loadingDeferred.done((items) => {
+      const eventArgs = {};
+
+      if (isSelectAll) {
+        eventArgs[isDeselect ? 'isDeselectAll' : 'isSelectAll'] = true;
+      }
```

Comment:
It's strange for me to see that eventArgs are created here.
The rest args are collected in [one place](https://github.com/DevExpress/DevExtreme/pull/29432/files#diff-68e40718c36f32e88e9c05867aa5e4935c0a16c1df8093833076a0723e142e03L93) based on `options`. Don't you think it's better to keep approach consistent? It'll allow to get rid of additional parameter in several places + reduce copypaste (now you create this eventArgs in 2 places)